src/components/MenuButton/MenuButton.component.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Styled from './styled';
```

Comment:
You could import elements with destructuring also
```suggestion
import { IconButton, MenuButton, SvgButton } from './styled';
```

Thus the usage would be cleaner in my opinion:
```
<MenuButton>
  <IconButton>
    <SvgButton />
  </IconButton>
</MenuButton>
```